package.json

```diff
@@ -1,6 +1,9 @@
 {
   "name": "Qowala-browser-plugin",
   "private": "true",
+  "dependencies": {
+    "vue": "^2.3.3"
```

Comment:
It would be better to add Vue as a simple JS file, like you did with Mithril, instead of as a node module. Because when I use `web-ext` to build the plugin, it omits the `node_modules` folder. 
That is understandable because it contains a lot of dev dependencies which should not be packaged for production. As it is complicated to package the plugin with only the Vue.js file from the node modules without adding all other dev dependencies, it would be nice to only have a simple JS file in the `lib/` folder :)